setup.py

```diff
@@ -46,7 +46,7 @@
             'pyro-api>=0.1.1'
         ],
         'dev': ['ipython', 'isort'],
-        'examples': ['matplotlib'],
+        'examples': ['matplotlib', 'seaborn'],
```

Comment:
I added seaborn here because matplotlib does not have a similar function for kdeplot.